src/handlers/shared/start.ts

```diff
@@ -83,7 +83,15 @@ export async function start(context: Context, issue: Context["payload"]["issue"]
   }
 
   const duration: number = calculateDurations(labels).shift() ?? 0;
-  const toAssignIds = toAssign.map(async (u) => await fetchUserId(context, u));
+  const toAssignIds = toAssign.map(async (u) => {
+    const userId = await fetchUserId(context, u);
+
+    if (!userId) {
+      throw new Error(logger.error("User not found", { user: u }).logMessage.raw);
+    }
+
+    return userId;
+  });
```

Comment:
I've just realised that this will return an array of promises, you have to call `Promise.all()` to get results